bootstrap/schema_ingester.py

```diff
@@ -77,16 +77,16 @@ def get_mixin_id(create_mixin_url, headers, mixin_title, data, class_id, tenant_
     data['description'] = mixin_title
     # Set the class id
     data['meta:intendedToExtend'][0] = class_id
+
     # Set the tenant id
-    for key in data["definitions"].keys():
+    for key in list(data["definitions"]):
         data["definitions"][mixin_definition_title] = data["definitions"][key]
-        for nested_key in data["definitions"][key]["properties"].keys():
+        for nested_key in list(data["definitions"][key]["properties"]):
             data["definitions"][key]["properties"][tenant_id] = data["definitions"][key]["properties"][nested_key]
             del data["definitions"][key]["properties"][nested_key]
-            del data["definitions"][key]
+        del data["definitions"][key]
```

Comment:
correct spacing here?

Reply:
Can you confirm the spacing on line 87?  Should it be nested?

Reply:
Yes it should not be nested, would throw an error when trying to delete.